code/processes/wdb.q

```diff
@@ -256,8 +259,9 @@ doreload:{[pt]
     /-inform gateway of reload start
     informgateway(`reloadstart;`);
     getprocs[;pt] each reloadorder;
-    if[eodwaittime>0;
+    $[eodwaittime>0;
         .timer.one[timeouttime::.proc.cp[]+eodwaittime;(value;".wdb.flushend[]");"release all hdbs and rdbs as timer has expired";0b];
+        .timer.one[.proc.cp[];(value;".wdb.flushend[]");"flush all hdbs and rdbs"; 0b]
```

Comment:
do we need to do this as a timer? can we not just call it directly?

```suggestion
        .wdb.flushend[]
```

Reply:
Apologies, I think I had misunderstood the purpose of having the timer function here